packages/web-components/@sfx/base/src/base.ts

```diff
@@ -0,0 +1,44 @@
+import { LitElement, customElement, html } from 'lit-element';
+import { makeSlot, createChildrenObserver } from './utils';
+
+@customElement('sfx-base')
+export default class Base extends LitElement {
+  observer: any;
+
+  constructor() {
+    super();
+    this.addSlots();
+  }
+
+  addSlots() {
+    this.shadowRoot.appendChild(makeSlot('before'));
+    this.shadowRoot.appendChild(makeSlot());
+    this.shadowRoot.appendChild(makeSlot('after'));
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.observer = createChildrenObserver(this);
+
+    this.observer.observe(this, {
+      childList: true,
```

Comment:
If this is the only option that we're going to pass, I would put this all in one line.